driver/gl/gl.go

```diff
@@ -213,28 +214,45 @@ func (c *glCanvas) newGlImageTexture(obj fyne.CanvasObject) uint32 {
 		}
 
 		if strings.ToLower(filepath.Ext(name)) == ".svg" {
-			icon, err := oksvg.ReadIconStream(file)
-			if err != nil {
-				log.Println("SVG Load error:", err)
-
-				return 0
+			rasterMutex.Lock()
+			defer rasterMutex.Unlock()
+			info := rasters[img.Resource]
+			if info == nil || info.w != width || info.h != height || info.alpha != img.Alpha() {
```

Comment:
Alpha is no longer needed here - we draw at full alpha to the bitmap and blend later in the gl details. So this can be simplified.